src/simulators/chart_simulator.ts

```diff
@@ -0,0 +1,33 @@
+import { message } from '../../src/shared/proto/messages'
+import { Simulator } from './simulator'
+import { Message } from './simulator'
+import DataPoint = message.support.nubugger.DataPoint
+
+export class ChartSimulator implements Simulator {
+  public static of() {
+    return new ChartSimulator()
+  }
+
+  public simulate(time: number): Message[] {
+    const messageType = 'message.support.nubugger.DataPoint'
+
+    const now = time
```

Comment:
oh right, I did that to save time on testing to make sure it worked, but forgot to update it.